core-services/workflow/src/main/java/org/silverpeas/core/workflow/engine/datarecord/ProcessInstanceDataRecord.java

```diff
@@ -23,8 +23,10 @@
  */
 package org.silverpeas.core.workflow.engine.datarecord;
 
+import org.silverpeas.core.ResourceReference;
 import org.silverpeas.core.contribution.content.form.Field;
 import org.silverpeas.core.contribution.content.form.FormException;
+import org.silverpeas.core.contribution.content.form.record.IdentifiedRecordTemplate;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.silverpeas.core.contribution.content.form.record.IdentifiedRecordTemplate'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)